src/backend/ref/dh-newhope.c

```diff
@@ -176,8 +176,10 @@ NoiseDHState *noise_newhope_new(void)
     return &(state->parent);
 }
 
+#if !USE_LIBSODIUM
 /* Implementation of random number generation needed by New Hope */
 void randombytes(unsigned char *x,unsigned long long xlen)
```

Comment:
Yes, libsodium has an API-compatible randombytes. I can just remove all this instead if you prefer.